2024.py

```diff
@@ -32,11 +32,11 @@
     nlp = spacy.load('en_core_web_md')
 
 # Check and install pyarrow
-try:
-    import pyarrow
-except ModuleNotFoundError:
-    print("pyarrow module not found. Installing...")
-    os.system('pip install pyarrow')
+# try:
```

Comment:
@TylerDollard We're not using this pyarrow library, so don't bother importing it.